app/(dashboard)/mailboxes/[mailbox_slug]/[category]/list/savedReplySelector.tsx

```
@@ -0,0 +1,112 @@
+import { MessageSquareText as SavedReplyIcon, ChevronDown, Search } from "lucide-react";
```

ESLint

[error] Replace `MessageSquareText·as·SavedReplyIcon,·ChevronDown,·` with `ChevronDown,·MessageSquareText·as·SavedReplyIcon,·`

(prettier/prettier)